setup.py

```diff
@@ -1,17 +1,54 @@
 import setuptools
+import sys
+
+if sys.version_info < (3, 7, 0):
+    raise OSError(f'Jina requires Python 3.7 and above, but yours is {sys.version}')
```

Comment:
We shouldn't worry about this, since we only use this as a front-end. No Jina code is involved. So we should tie it to whatever version of Streamlit is important